src/libConsensus/ConsensusLeader.cpp

```diff
@@ -894,6 +894,41 @@ bool ConsensusLeader::ProcessMessage(const bytes& message, unsigned int offset,
   return result;
 }
 
+void ConsensusLeader::Audit() {
+  LOG_MARKER();
+
+  lock_guard<mutex> g(m_mutex);
+
+  for (unsigned int subsetID = 0; subsetID < m_consensusSubsets.size();
+       subsetID++) {
+    ConsensusSubset& subset = m_consensusSubsets.at(subsetID);
+
+    if ((subset.state == CHALLENGE_DONE) ||
+        (subset.state == FINALCHALLENGE_DONE)) {
+      if (subset.commitMap.size() != m_committee.size()) {
```

Comment:
Is subset commitMap size expected to be same as m_commitee?

Reply:
Yes, since the map represents each member in the committee (see `GenerateConsensusSubsets`)